src/main/resources/templates/map-location.html

```diff
@@ -65,7 +65,7 @@ <h1 class="govuk-heading-xl mv2" th:inline="text">Update location</h1>
             <button class="govuk-button mv3 mr3" data-module="govuk-button">
                 Confirm and save
             </button>
-            <a href="/dashboard" class="govuk-link">Cancel</a>
+            <a href="/journeys" class="govuk-link">Cancel</a>
```

Comment:
This template serves as both update and create so this link may need to smarter in terms of where it goes...